api-contract/src/main/java/uk/gov/hmcts/payment/api/contract/util/PaymentProviders.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.payment.api.contract.util;
+
+public enum PaymentProviders {
```

Comment:
We already keep all the Payment Channels/Providers/Methods in DB, and access through the models at package uk.gov.hmcts.payment.api.model;
Can we not utilise them here.

Reply:
Reused the existing class and deleted the new one.